README.md

```diff
@@ -8,7 +8,7 @@
 
 Capacitor plugin to support [Sign in With Apple](https://developer.apple.com/sign-in-with-apple/get-started/)
 
-> ### :rotating_light: This plugin is for Capacitor 5 :rotating_light:
+> ### :rotating_light: This plugin is for Capacitor >= 5 :rotating_light:
```

Comment:
Actually the 6.x version is for Capacitor 6

Reply:
@piotr-cz Yes, but this also does support the v5, therefore `>= 5` if I'm not wrong

Reply:
No, capacitor 5 users should use 5.x, 6.x is for capacitor 6 only. I’ve merged another PR that documented it adding info about 5.x, so closing yours.